lib/__tests__/html-compiler.test.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+const tempy = require('tempy');
+const path = require('path');
+const fs = require('fs');
+
+jest.mock('uglify-js', () => ({
+  minify: jest.fn()
+}));
+
+const minifyMock = require('uglify-js').minify;
+
+const { writeHtml, jsBundles, polyfill } = require('../html-compiler');
+
+test('jsBundles', () => {
```

Comment:
What are we testing here about `jsBundles`?

Reply:
Whoops, forgot to describe it.